.gitpod.Dockerfile

```
@@ -0,0 +1,21 @@
+USER gitpod
+
+# We use latest major version of Node.js distributed VS Code. (see about dialog in your local VS Code)
+RUN bash -c ". .nvm/nvm.sh \
+    && nvm install 20 \
+    && nvm use 20 \
+    && nvm alias default 20"
+
+RUN echo "nvm use default &>/dev/null" >> ~/.bashrc.d/51-nvm-fix
+
+# Install dependencies
+RUN sudo apt-get update \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)